activities/communication/pitching.md

```diff
@@ -61,8 +61,8 @@ We have different perspectives on open source:
 
 We should reflect our [founding principles](https://about.publiccode.net/organization/mission.html#founding-principles) in our pitches.
 
-Avoid: "we think or we believe" [open source is better]
-Instead, describe what we do: "we help municipalities [share collaborative software].
+Avoid: "we think or we believe" [open source is better].
+Instead, describe what we do: "we help public organizations [share collaborative software].
```

Comment:
This is of course technically more correct, but I often like to mention municipalities as it is less abstract and is often easy for people to relate to.

Reply:
It the pitching guidance for printed text as well? I was thinking that it was more about verbal, but maybe that is wrong. In printed text, I'd lean  be more technically correct and in engaged conversation be more loose and prefer something most easy to absorb.

Reply:
reverted